ShopifySharp/Services/Multipass/MultipassService.cs

```diff
@@ -0,0 +1,99 @@
+﻿using Newtonsoft.Json;
+using System;
+using System.IO;
+using System.Linq;
+using System.Security.Cryptography;
+using System.Text;
+
+namespace ShopifySharp
+{
+    public class MultipassService
```

Comment:
It looks like the only method on this class is `GetMultipassUrl`, would it make more sense to make the class static, remove the constructor, and then make the method take `myShopifyUrl` and `multipassSecret` in addition to `customer`?

Reply:
Hey, I have updated as your asked.